test/test_tomopy/test_misc/test_corr.py

```diff
@@ -109,6 +111,33 @@ def test_median_filter_nonfinite(self):
                     size=3,
                     callback=None,
                 )
+    
+    def test_median_filter3d(self):
+        # generate a small noncubic volume and apply random noise
+        A = np.float32(np.zeros((30,40,20)))
+        A[:] = 0.1
+        A[10:20,20:30,:] = 0.5
+        np.random.seed(1)
+        A_noise = A + np.random.normal(loc=0,
+                          scale=0.03 * A,
+                          size=np.shape(A))
+        A_filtered = median_filter3d(np.float32(A_noise))
+        max_filtered = 0.513833
+        assert_allclose(np.max(A_filtered), max_filtered, rtol=1e-06)
```

Comment:
How do you know that the maximum value of the filtered array is 0.513833 if the noise is random?

Can we compare with a human verifiable result instead. For example:

```
A = np.arange(4*5*6).reshape(4,5,6)
A1 = median_filter3d(A, 1)
np.testing.assert_equal(A, A1)
A3 = median_filter3d(A, 3)
np.testing.assert_equal(truth, A3)

```

Reply:
many thanks, it is clearer now! I will go and do the checking as we might need a specific symmetric boundary condition in _scipy.ndimage.median_filter_ to fit the implementation.

Reply:
done in [51566bb](https://github.com/tomopy/tomopy/pull/596/commits/51566bb7091d8cbc3fb9b1f0b524a357c3063ebb)